packages/penrose-core/src/compiler/Style.ts

```diff
@@ -2665,20 +2705,26 @@ export const compileStyle = (
     labelMap,
     styVals
   );
-  // TODO(error return) ^
 
   log.info("translation (before genOptProblem)", translateRes);
 
-  // Translation failed somewhere. // TODO: Check that errors are returned in a consistent order
+  // Translation failed somewhere
   if (translateRes.tag === "Left") {
-    // const err = `Could not compile. Error(s) in Style while generating translation`;
-    return err(genericStyleError(translateRes.contents));
+    return err(firstStyleError(translateRes.contents));
```

Comment:
Not sure why you are reporting just the first error. Any reason? You could do `interface StyleErrorList { errors: StyleError[] }` as a branch of `StyleError`.